jetbrains-core/src/software/aws/toolkits/jetbrains/core/region/AwsRegionProvider.kt

```diff
@@ -1,9 +1,11 @@
-// Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 
 package software.aws.toolkits.jetbrains.core.region
 
 import com.intellij.openapi.components.ServiceManager
+import com.jetbrains.rd.util.firstOrNull
```

Comment:
You pulled in RD here, avoid RD directly right now

Reply:
we should prolly add a lint rule for this